packages/web3-shared/evm/hooks/useAssets.ts

```diff
@@ -1,12 +1,13 @@
-import type { ChainId, FungibleTokenDetailed } from '../types'
+import type { Asset, ChainId, FungibleTokenDetailed } from '../types'
 import { useWallet } from './useWallet'
 import { useNativeTokenDetailed } from './useNativeTokenDetailed'
 import { useAssetsFromChain } from './useAssetsFromChain'
 import { useAssetsFromProvider } from './useAssetsFromProvider'
-import { useCallback } from 'react'
+import { useCallback, useMemo } from 'react'
 import { useAssetsMerged } from './useAssetsMerged'
 import { formatEthereumAddress } from '../utils'
 
+const emptyList: Asset[] = []
```

Comment:
Not necessary.

Reply:
Yes, I plan to replace all inline array in the code base with a stable array, that will reduce re-renderings.

Reply:
I see. Better to name them `EMPTY_LIST`.

Reply:
I will.

Reply:
Will refactor EMPTY_LIST in another PR.